examples/bite_sized/dynamic_output.py

```diff
@@ -0,0 +1,72 @@
+import csv
+import re
+import os
+import xml.etree.ElementTree as ET
+
+import requests
+import pandas as pd
+
+from dagster import (
+    Out,
+    Output,
+    job,
+    op,
+    resource,
+    DynamicOut,
+    DynamicOutput,
+    io_manager,
+    OutputContext,
+)
+
+from bite_sized.conditional_branching import DataframeToCSVIOManager
```

Comment:
the idea i had in mind was to make every single example to be standalone so users can just look at one repo/job at a time and be able to execute runs just using that example. wonder how you feel about this idea?
see more discussions on https://github.com/dagster-io/dagster/pull/5787